portal-web/docroot/html/taglib/ui/input_date/page.jsp

```diff
@@ -46,7 +46,18 @@ String monthParamId = namespace + HtmlUtil.getAUICompatibleId(monthParam);
 String nameId = namespace + HtmlUtil.getAUICompatibleId(name);
 String yearParamId = namespace + HtmlUtil.getAUICompatibleId(yearParam);
 
-Calendar calendar = CalendarFactoryUtil.getCalendar(yearValue, monthValue, dayValue);
+Calendar calendar = null;
+
+if (required) {
+	calendar = CalendarFactoryUtil.getCalendar(timeZone);
+
+	yearValue = calendar.get(Calendar.YEAR);
+	monthValue = calendar.get(Calendar.MONTH);
+	dayValue = calendar.get(Calendar.DAY_OF_MONTH);
+}
+else {
+	calendar = CalendarFactoryUtil.getCalendar(yearValue, monthValue, dayValue);
+}
```

Comment:
This does not look right. If `*Value` attributes are set, but also `required` is true, `*Value` params will be ignored and we will take current value.